bpf/configs.h

```diff
@@ -6,5 +6,6 @@
 volatile const u32 sampling = 0;
 volatile const u8 trace_messages = 0;
 volatile const u8 enable_rtt = 0;
+volatile const u64 min_rtt = 50000; //50 micro seconds
```

Comment:
are u planning to make this configurable ? because I don't see the userspace pieces to do that if not then u can define in it flows.c where its actually used and no need to make volatile in that case